demo/cmd/all/main.go

```diff
@@ -36,9 +36,10 @@ func main() {
 		},
 		EnableDebug: *debug,
 	}
-	subgraphs, err := subgraphs.New(&config)
+	ctx := context.Background()
+	subgraphs, err := subgraphs.New(ctx, &config)
```

Comment:
Wouldn't it make more sense to create a ctx in the testenv, which we cancel using textenv cleanup:
https://github.com/wundergraph/cosmo/blob/5643cc9268f9f7b3fc3de86eaf7f3da11178e660/router-tests/testenv/testenv.go#L68

Reply:
It is not for test env, it is cmd in demo to run all subgraphs

Test env already uses context which it propagates to subgraphs
https://github.com/wundergraph/cosmo/blob/7616e360047dd4ab183c1ddf835a079747ff1e50/router-tests/testenv/testenv.go#L162